docs/resources/bms_instance.md

```diff
@@ -0,0 +1,217 @@
+    from My Credential on the management console. Changing this creates a new instance.
+
+* `user_data` - (Optional, String, ForceNew) Specifies the user data to be injected during the instance creation.
+    Text and text files can be injected.
+    Changing this creates a new server.
+
+* `admin_pass` - (Optional, String, ForceNew) Specifies the administrative password to assign to the instance.
+    Changing this creates a new instance.
+
+* `key_name` - (Optional, String, ForceNew) Specifies the name of a key pair to put on the instance.
+    The key pair must already be created and associated with the tenant's account.
+    Changing this creates a new instance.
+
+* `security_groups` - (Optional, List, ForceNew) Specifies a array of one or more security group ids to associate
+    with the instance.
+
+* `availability_zone` - (Required, String, ForceNew) Specifies the availability zone in which to create
+    the instance. Please following [reference](https://developer.huaweicloud.com/intl/en-us/endpoint?BMS)
+    for the values. Changing this creates a new instance.
+
+* `vpc_id` - (Required, List, ForceNew) Specifies id of vpc in which to create the instance.
+
+* `nics` - (Required, List, ForceNew) Specifies an array of one or more networks to attach to the instance.
+    The network object structure is documented below.
+    Changing this creates a new instance.
+
+* `eip_id` - (Optional, String, ForceNew) The ID of the EIP. Changing this parameter will create a new resource.
+
+
+-> **Note:** If the eip_id parameter is configured, you do not need to configure the bandwidth parameters:
+  `iptype`, `eip_charge_mode`, `bandwidth_size`, `share_type` and `bandwidth_charge_mode`.
+
+* `iptype` - (Optional, String, ForceNew) Elastic IP type. Changing this parameter will create a new resource.
+
+* `eip_charge_mode` - (Optional, String, ForceNew) Elastic IP billing type. Changing this parameter will create a new resource.
+
+* `sharetype` - (Optional, String, ForceNew) Bandwidth sharing type. Changing this parameter will create a new resource.
+
+* `bandwidth_size` - (Optional, Int, ForceNew) Bandwidth size. Changing this parameter will create a new resource.
+
+* `bandwidth_charge_mode` - (Optional, String, ForceNew) Bandwidth billing type. Changing this parameter will create a new resource.
+
+
+
+* `system_disk_type` - (Optional, String, ForceNew) Specifies the system disk type of the instance.
+  For details about disk types, see
+  [Disk Types and Disk Performance](https://support.huaweicloud.com/en-us/productdesc-evs/en-us_topic_0014580744.html)
+    Available options are:
+	* `SSD`: ultra-high I/O disk type.
+	* `GPSSD`: general purpose SSD disk type.
+	* `SAS`: high I/O disk type.
+    Changing this creates a new instance.
+
+* `system_disk_size` - (Optional, int, ForceNew) Specifies the system disk size in GB, The value range is 1 to 1024.
```

Comment:
The minimum of the system disk size should be 40.

Reply:
You can check the console. Accturaly, the value of size ranges from 0 to 1024. But I think 0 is useless.

Reply:
The system disk size must be greater than or equal to the minimum system disk size of the image. I added this description.